x-pack/plugins/aiops/public/application/url_state/log_pattern_analysis.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getDefaultAiOpsListState, type AiOpsFullIndexBasedAppState } from './common';
+
```

Comment:
This file was broke out from above [application/url_state/common.ts](https://github.com/elastic/kibana/pull/171398/files#diff-74895863978e3c12e22dd502de4b9dfa90047eded6e436e984fe411c02ec09f0)